modules/AOR_Conditions/AOR_Condition.php

```diff
@@ -95,6 +95,9 @@ function save_lines($post_data, $parent, $key = '')
         require_once('modules/AOW_WorkFlow/aow_utils.php');
 
         $j = 0;
+        if(!isset($post_data[$key . 'field']) || !is_array($post_data[$key . 'field'])){
+            return;
```

Comment:
Can you please add a warning to screen and/or crm log at least, to see why the function failed, it would be much easier to debugging in future.
for e.g: ```
global $log;
$log->warn('Something important information to see what just happened here...');
```

Reply:
In this case the variable in question not being set / not being an array happens when a report is created that doesn't contain any condition lines.

In itself that is not an error, so I'm not sure there should be any logging in this particular case.